code/modules/mob/living/simple_animal/bot/firebot.dm

```diff
@@ -14,7 +14,7 @@
 	health = 25
 	maxHealth = 25
 
-	maints_access_required = list(ACCESS_ROBOTICS, ACCESS_CONSTRUCTION)
+	req_access = list(ACCESS_ROBOTICS, ACCESS_CONSTRUCTION)
```

Comment:
All of these should be `req_one_access`, not `req_acccess`, according to the old code